### ISSUE TITLE
[Feature]  lr shold scale when set accumulative_counts > 1?

### ISSUE THREAD

**Issue by jeannotes** [open-mmlab/mmengine#894]
### What is the feature?

hi, I am wondering when set accumulative_counts = 2, this means the batchsize scale up 2X, and the lr should also scale up?
thanks for your help

**Comment by zhouzaida** [comment]
There are two cases:

- The optimal `batch size` and `lr` have been determined and you just want to increase the batch size to shorten the time, then you should expand the learning rate at the same time
- The optimal `batch size` given in the paper is N, when your memory can only train N/2 samples, then the learning rate does not need to be adjusted.
